lib/index.js

```diff
@@ -16,6 +17,7 @@ var omit = require('omit');
 var Wootric = module.exports = integration('Wootric')
   .assumesPageview()
   .option('accountToken', '')
+  .global('event_name')
```

Comment:
Hey wootric team - is `event_name` a global variable set by the sdk?

Reply:
`event_name` is set by `track()`. So I'm guessing this doesn't go here? hehe